% FortySecondsCV LaTeX template
% Copyright © 2019-2022 René Wirnata <rene.wirnata@pandascience.net>
% Licensed under the 3-Clause BSD License. See LICENSE file for details.
%
% Please visit https://github.com/PandaScience/FortySecondsCV for the most
% recent version! For bugs or feature requests, please open a new issue on
% github.
%
% Contributors:
% https://github.com/PandaScience/FortySecondsCV/graphs/contributors
%
% Attributions
% ------------
% * fortysecondscv is based on the twentysecondcv class by Carmine Spagnuolo
%   (cspagnuolo@unisa.it), released under the MIT license and available under
%   https://github.com/spagnuolocarmine/TwentySecondsCurriculumVitae-LaTex
% * further attributions are indicated immediately before corresponding code


%-------------------------------------------------------------------------------
%                             ADDITIONAL PACKAGES
%-------------------------------------------------------------------------------
\documentclass[
	a4paper,
	% 9pt,
	% sidesectionsize=Large,
	% showframes,
	% vline=2.2em,
	% maincolor=cvgreen,
	% sidecolor=gray!50,
	% sidetextcolor=green,
	% sectioncolor=red,
	% subsectioncolor=orange,
	% itemtextcolor=black!80,
	% sidebarwidth=0.4\paperwidth,
	% topbottommargin=0.03\paperheight,
	% leftrightmargin=20pt,
	% profilepicsize=4.5cm,
	% profilepicborderwidth=3.5pt,
	% profilepicstyle=profilecircle,
	% profilepiczoom=1.0,
	% profilepicxshift=0mm,
	% profilepicyshift=0mm,
	% profilepicrounding=1.0cm,
	% logowidth=4.5cm,
	% logospace=5mm,
	% logoposition=before,
	% sidebarplacement=right,
	% datecolwidth=0.22\textwidth,
]{fortysecondscv}

% fine tune line spacing
% \usepackage{setspace}
% \setstretch{1.1}

% improve word spacing and hyphenation
\usepackage{microtype}
\usepackage{ragged2e}
\usepackage{enumitem}

% uncomment in case you don't want any hyphenation
% \usepackage[none]{hyphenat}

% take care of proper font encoding
\ifxetexorluatex
	\usepackage{fontspec}
	\defaultfontfeatures{Ligatures=TeX}
	% \newfontfamily\headingfont[Path=fonts/]{segoeuib.ttf} % use local font
\else
	\usepackage[utf8]{inputenc}
	\usepackage[T1]{fontenc}
\fi

% use a sans serif font as default
\usepackage[sfdefault]{ClearSans}
% \usepackage[sfdefault]{noto}

% multi-language CV XeLaTeX and polyglossia (should also work with LuaLaTeX)
% NOTE: breaks \pointskill, \membership and some spacings
% \ifxetexorluatex
% 	\usepackage{polyglossia}
% 	\newfontfamily\arabicfontsf[Script=Arabic,Scale=1.5]{Amiri}
% 	\newfontfamily\englishfontsf{Clear Sans}
% 	\setmainfont{Amiri}
% 	\setdefaultlanguage{arabic}
% 	\setotherlanguage{english}
% \fi

% enable mathematical syntax for some symbols like \varnothing
\usepackage{amssymb}


% bubble diagram configuration
\usepackage{smartdiagram}
\smartdiagramset{
	% default font size is \large, so adjust to harmonize with sidebar layout
	bubble center node font = \footnotesize,
	bubble node font = \footnotesize,
	% default: 4cm/2.5cm; make minimum diameter relative to sidebar size
	bubble center node size = 0.4\sidebartextwidth,
	bubble node size = 0.25\sidebartextwidth,
	distance center/other bubbles = 1.5em,
	% set center bubble color
	bubble center node color = maincolor!70,
	% define the list of colors usable in the diagram
	set color list = {maincolor!10, maincolor!40,
	maincolor!20, maincolor!60, maincolor!35},
	% sets the opacity at which the bubbles are shown
	bubble fill opacity = 0.8,
}

%-------------------------------------------------------------------------------
%                            PERSONAL INFORMATION
%-------------------------------------------------------------------------------
%% mandatory information
% your name

\cvname{\centering{\huge{Sébastien Gradit}}}
% job title/career
% AFTER: Reduces the space after the job title for a tighter look
\cvjobtitle{\centering{Computational Biologist Data Scientist}\\[0.25pt]\vspace{-2pt}}

%% optional information
% profile picture
% \cvprofilepic{pics/profile.png} % Profile picture
% logo picture
% \cvlogopic{pics/logo_txt.png}

% NOTE: ordering in sidebar will mimic the following order
% date of birth
% \cvbirthday{\today} % Birth date
% short address/location, use \newline if more than 1 line is required
% \cvaddress{\small{Issy-les-Moulineaux, France}}
% % phone number
% \cvphone{\small{+33 6 50 84 94 62}}
% % personal website
% \cvsite{\small{https://sebastiengradit.com}}
% % email address
% \cvmail{\small{sebastiengradit@gmail.com}}
\cvaddress{\small{Issy-les-Moulineaux}}
\cvphone{\small{+33 6 50 84 94 62}}
\cvmail{\small{sebastiengradit@gmail.com}}
\cvsite{\small{https://sebastiengradit.com}}
\cvgithub{\small{sebgra}}
\cvlinkedin{\small{sebastien-gradit}}

% pgp key
% \cvkey{4096R/FF00FF00}{0xAABBCCDDFF00FF00} % Change for ORCID ? 
% any other custom entry
% \cvcustomdata{\faFlag}{Chinese} % Flag for nationality

%-------------------------------------------------------------------------------
%                              SIDEBAR 1st PAGE
%-------------------------------------------------------------------------------
% add more profile sections to sidebar on first page
\addtofrontsidebar{
	% \vspace{-1em} % reduce space above first section
	% include gosquare national flags from https://github.com/gosquared/flags;
	% naming according to ISO 3166-1 alpha-2 country codes
	\graphicspath{{pics/gosquared-flags/flags/flags-iso/shiny/64}}

	% social network accounts incl. proper hyperlinks
	% \sidesection{Social Network}
	% \begin{tabular}{p{0.55\linewidth} p{0.55\linewidth}}
	% 	\begin{icontable}{2em}{1em}
	% 		% \social{\faLeaf}
	% 		% 	{https://de.overleaf.com/latex/templates/forty-seconds-cv/pztcktmyngsk}
	% 		% 	{Overleaf Template Link}
	% 		\social{\faLinkedin}
	% 			{https://linkedin.com/in/sebastien-gradit}
	% 			{S. Gradit}
	% 	\end{icontable} 
	% 	&
	% 	\begin{icontable}{2em}{1em}
	% 		% \social{\faLeaf}
	% 		% 	{https://de.overleaf.com/latex/templates/forty-seconds-cv/pztcktmyngsk}
	% 		% 	{Overleaf Template Link}
	% 		\social{\faGithub}
	% 			{https://github.com/sebgra}
	% 			{sebgra}
	% 	\end{icontable}
	% \end{tabular}

	\sidesection{\large About Me}
	\vspace{0em} % Reducing space below the title
		\aboutme{
		\small{Highly accomplished PhD Computational Biologist with over five years of experience, specializing in bridging
		AI/ML solutions and Bioinformatics to transform complex multi‑omics data into actionable insights for R\&D.
		My expertise includes advanced computational methods and data‑driven approaches to accelerate tool deployment and integration, with a strong focus on reproducibility and multidisciplinary collaboration.}
		}

	\sidesection{\large{Languages}}
		% \pointskill{\flag{CN.png}}{Chinese}{5}
	% 	\pointskill{\flag{DE.png}}{German}{3}
	% \pointskill{\flag{GB.png}}{English}{3}
	% \pointskill{\flag{FR.png}}{French}{3}
		\setlength{\leftmargini}{0.2cm}
		\begin{itemize}
			\item English: Fluent (TOEIC 955/990)
			% \item French: Mother tongue
		\end{itemize}

\sidesection{\Large{Skills}}
\vspace{0em} 
\setlength{\leftmargini}{0.2cm}

\small{
    \begin{itemize}
        \raggedright % <-- ADDED: Disables justification and thus hyphenation for better spacing
        
        \item \textbf{Core Programming:} Bash - Python [advanced], C++ [Intermediate], R [Intermediate], High Performance Computing [Slurm], Cloud [AWS], Versioning [Git, GitHub]
        
        \item \textbf{Data Science / ML:} Machine Learning [Keras, TensorFlow, PyTorch, SciKit-Learn], SQL [Intermediate], Interactive Dashboards [Streamlit, Shiny]
        
        \item \textbf{Data Management / Workflow:} Continuous Integration/Continuous Development [GitHub Actions, Snakemake, Makefile], Containerization [Docker, Singularity]
        
        \item \textbf{Bioinformatics:} Package Development [PyPI, Conda], Multi-Omics Integration, Automation of Mapping and Downstream Analyses (NGS), High-Throughput Omics Data Analysis [RNA-Seq, ChIP-Seq, Hi-C, Single-cell Omics]
        
    \end{itemize}
} % End the font size change here
		% \skill{\faBalanceScale}{Sleeping almost all day}
		% \skill{\faSitemap}{Eating a lot of bamboo sprouts}
		% \skill{\faGraduationCap}{Relaxing rest of the day}

	%% Double column soft skill section

	% \sidesection{\Large{Soft Skills}}
	% \vspace{-2em}
	% 	% \pointskill{\faHome}{Looking Cute}{4}[4]
	% 	% 	\skill[1.8em]{\faCompress}{No need to specify further}
	% 	% \pointskill{\faChild}{Chillin' hard}{3}[4]
	% 	% 	\skill[1.8em]{\faCompress}{On a tree}
	% 	% 	\skill[1.8em]{\faCompress}{In the grass}
	% 	\begin{tabular}{p{0.55\linewidth} p{0.55\linewidth}} % Double sided list
	% 		\begin{itemize}
	% 			\item Analytical Thinking \&Complex Problem‑Solving
	% 			\item Adaptability, Agility \& Continuous Learning
	% 		\end{itemize} &
	% 		\begin{itemize}
	% 			\item Creative Thinking \& Innovation
	% 			\item Effective Communication \& Interdisciplinary Collaboration
	% 		\end{itemize} 
	% 	  \end{tabular}

		\sidesection{\large{Soft Skills}}
		\vspace{-2em}
		% \pointskill{\faHome}{Looking Cute}{4}[4]
		% 	\skill[1.8em]{\faCompress}{No need to specify further}
		% \pointskill{\faChild}{Chillin' hard}{3}[4]
		% 	\skill[1.8em]{\faCompress}{On a tree}
		% 	\skill[1.8em]{\faCompress}{In the grass}
		\begin{tabular}{p{1.0\linewidth}} % Double sided list
			\begin{itemize}
				\item \small{Analytical Thinking, Complex Problem-Solving}
				\item \small{Adaptability, Agility, Continuous Learning}
			
				\item \small{Creative Thinking, Innovation}
				\item \small{Effective Communication, Interdisciplinary Collaboration}
			\end{itemize} 
		  \end{tabular}

\sidesection{\Large{Extracurricular Activities}}
\vspace{1em} % Reduce space below the title
\begin{itemize}
    \setlength{\itemsep}{0pt} % Zero space between list items
    \setlength{\parskip}{0pt}  % Zero space between paragraphs
    \setlength{\leftmargin}{0pt} % Remove left indentation for the list

    \item \small{\textbf{JeBiF PRESIDENT (2022–24):} Led bioinformatics talent structure/energy; Fostered N/I collaborations; Promoted/popularized bioinformatics; Organized community events.}
    
\end{itemize}
}


%-------------------------------------------------------------------------------
%                              SIDEBAR 2nd PAGE
%-------------------------------------------------------------------------------
\definecolor{pastelgreen}{HTML}{D7ECD9}
\definecolor{pastelpurple}{HTML}{D5D6EA}
\definecolor{pastelorange}{HTML}{F5D5CB}
\definecolor{pastelyellow}{HTML}{F6F6EB}

\addtobacksidebar{
	\sidesection{About Me}
	\aboutme{
		PhD-trained bioinformatician from the Pasteur Institute seeking to apply expertise in data science and biological interpretation to advance research.
		Proven ability to leverage machine learning and statistical modeling to solve complex biological problems.
	}

	\sidesection{Diagrams}
	\begin{sidebarminipage}
		\chartlabel[pastelgreen]{Bubble}
		\chartlabel[pastelgreen]{Diagrams}
		\chartlabel[pastelpurple]{with}
		\chartlabel[pastelpurple]{proper}
		\chartlabel[pastelorange]{overflow}
		\chartlabel[pastelorange]{protection}
		\chartlabel[pastelyellow]{for}
		\chartlabel[pastelyellow]{labels}
	\end{sidebarminipage}

	\begin{figure}\centering
		\smartdiagram[bubble diagram]{
			\textcolor{white}{\textbf{Being a}} \\
			\textcolor{white}{\textbf{Panda}}, % center bubble
			\textcolor{black!90}{Eating},
			\textcolor{black!90}{Sleeping},
			\textcolor{black!90}{Rolling},
			\textcolor{black!90}{Playing},
			\textcolor{black!90}{Chilling}
		}
	\end{figure}

	\chartlabel{Wheel Chart}

	\wheelchart{3.7em}{2em}{%
	20/3em/maincolor!50/Chill,
	15/3em/maincolor!15/Play,
	30/4em/maincolor!40/Sleep,
	20/3em/maincolor!20/Eat
	}

	\sidesection{Barskills}
	\barskill[1ex]{\faSkyatlas}{Wearing asian rice hats}{60}
	\barskill[2ex]{\faImage}{Playing Chess}{30}
	\barskill[3ex]{\faMusic}{Playing the bamboo flute}{50}

	\sidesection{Memberships}
	\begin{memberships}
		\membership[4em]{pics/logo.png}{PandaScience.net}
		\membership[4em]{pics/logo.png}{Some longer text spanning over more than
			only one line}
	\end{memberships}
}


%-------------------------------------------------------------------------------
%                         TABLE ENTRIES RIGHT COLUMN
%-------------------------------------------------------------------------------
\begin{document}

\makefrontsidebar

\cvsection{Working Experience}
\vspace{-9pt}
\begin{cvtable}[1.75]
    \cvitem{2021-2024}{Bioinformatician}{Pasteur Institute, Paris}{
        \begin{itemize}[
            leftmargin=1pt, 
            topsep=0pt,   
            partopsep=0pt,
            itemsep=-2pt,
            after=\vspace{-2pt},
            label=\textbullet 
            % REMOVED: align=\justified
        ]
            \justifying % <<< ADD THIS HERE
            \item  Led comprehensive PhD research focused on deciphering hidden chromatin contacts from repeated genomic elements via advanced statistical profiling.
            \item Developed Hicberg, a novel computational tool for reconstructing missing chromatin contacts in repeated regions, extensible to pair‑ended
					omics data, leading to new insights into the spatial organization of genomes.
			\item  Applied Hicberg to investigate Saccharomyces cerevisiae rDNA behavior and chromatin dynamics under various stress conditions (e.g., heat,
					oxidative) and kinetics.
			\item  Leveraged Deep Learning to predict 2‑micron plasmid interactions with the yeast genome based on nucleosome occupancy data.
        \end{itemize}
    }

	 \cvitem{02/2020 -- 08/2020}{Biomedical Engineer}{Siemens Healthcare - LIPADE, Paris}{
        \begin{itemize}[
            leftmargin=1pt, 
            topsep=0pt,   
            partopsep=0pt,
            itemsep=-2pt,
            after=\vspace{-2pt},
            label=\textbullet 
            % REMOVED: align=\justified
        ]
            \justifying % <<< ADD THIS HERE
            \item  Spearheaded the development of Machine Learning‑driven automatic segmentation tools for bone lesion identification in (x)SPECT volumes, significantly enhancing diagnostic workflow efficiency and providing radiologists with data‑driven insights for more confident decision‑
					making in oncology.
			\item  Engineered an end‑to‑end pipeline for automated extraction of radiomic features from xSPECT volumes, accelerating objective region of
				interest classification and segmentation to deliver critical, actionable data for diagnostic support.
        \end{itemize}
    }

	 \cvitem{02/2019 -- 09/2019}{Biomedical Engineer}{Danone Research, Saclay}{
        \begin{itemize}[
            leftmargin=1pt, 
            topsep=0pt,   
            partopsep=0pt,
            itemsep=-2pt,
            after=\vspace{-2pt},
            label=\textbullet 
            % REMOVED: align=\justified
        ]
            \justifying % <<< ADD THIS HERE
            \item  Pioneered the implementation and benchmarking of Area Under the Curve(AUC) computation techniques for bioequivalence assays, optimizing
			data reliability.
			\item Engineered an interactive user interface for bioequivalence assay analysis, enabling real‑time computation, visualization, and comparative
			insights.
			\item Integrated Machine Learning models to refine clinical trial plans, directly supporting data‑driven decision‑making
        \end{itemize}
    }

	\cvitem{05/2018 -- 08/2018}{Biomedical Engineer}{CABOMA - UdeM, Montreal}{
        \begin{itemize}[
            leftmargin=1pt, 
            topsep=0pt,   
            partopsep=0pt,
            itemsep=-2pt,
            after=\vspace{-2pt},
            label=\textbullet 
            % REMOVED: align=\justified
        ]
            \justifying % <<< ADD THIS HERE
            \item Optimized biomechanical models for patient morphology tailored orthotic design.
        \end{itemize}
    }
\end{cvtable}

\vspace*{1pt}


\cvsection{Education}
% \cvsubsection{Postgraduate Training}
% \begin{cvtable}[1.5]
% 	\cvitem{2009 -- 2010}{Post-Doc Panda Studies}{Panda Academy}
% 		{In-depth studies on the impact of bamboo nutrition for young pandas and
% 		its relation to relaxing, sleeping and snoozing parts of the day.}
% 	\cvitem{2008 -- 2009}{Research Stay Europe}{European Panda Labs}
% 		{Spending one year abroad teaching european panda facilities about the
% 		newest findings and research in the field of asian rice hat covers and
% 		applications for bamboo as a material.}
% \end{cvtable}


% \cvsubsection{Study}

%% Old version:
% \begin{cvtable}[2]
% 	\cvitem{2021 -- 2024}{PhD in bioinformatics}{Pasteur Institute - Sorbonne University}
% 		{Regulation Spatiale des Genomes lab.}
% 	\cvitem{2019 -- 2020}{M.Sc. in Image Processing}{Télécom Paris - Sorbonne University}
% 		{Advanced medical images processing and A.I. | \textit{Honors}}
% 	\cvitem{2016 -- 2019}{Master's Degree in Engineering (Dipl. Ing.)}{ISBS - ESIEE Paris}
% 		{Bioinformatics, Medical Imagine and A.I. | \textit{Highest Honors}}
% \end{cvtable}

%% New version:
\vspace{-8pt}
\begin{cvtable}[1.3]
    \cvitem{2021-2024}{PhD in Bioinformatics}{Pasteur Institute, Sorbonne University}{}
    %     \begin{itemize}[
    %         leftmargin=1pt, 
    %         topsep=0pt,   
    %         partopsep=0pt,
    %         itemsep=-2pt,
    %         after=\vspace{-2pt},
    %         label=\textbullet 
    %         % REMOVED: align=\justified
    %     ]
    %         \justifying % <<< ADD THIS HERE
    %         \item  Led comprehensive PhD research focused on deciphering hidden chromatin contacts from repeated genomic elements via advanced statistical profiling.
    %         \item Developed Hicberg, a novel computational tool for reconstructing missing chromatin contacts in repeated regions, extensible to pair‑ended
	% 				omics data, leading to new insights into the spatial organization of genomes.
	% 		\item  Applied Hicberg to investigate Saccharomyces cerevisiae rDNA behavior and chromatin dynamics under various stress conditions (e.g., heat,
	% 				oxidative) and kinetics.
	% 		\item  Leveraged Deep Learning to predict 2‑micron plasmid interactions with the yeast genome based on nucleosome occupancy data.
    %     \end{itemize}
    % }
	 \cvitem{2019-2020}{M.Sc. in Image Processing and A.I | \textit{Honors}}{Télécom Paris - Sorbonne University}{}
    %     \begin{itemize}[
    %         leftmargin=1pt, 
    %         topsep=0pt,   
    %         partopsep=0pt,
    %         itemsep=-2pt,
    %         after=\vspace{-2pt},
    %         label=\textbullet 
    %         % REMOVED: align=\justified
    %     ]
    %         \justifying % <<< ADD THIS HERE
    %         \item  Led comprehensive PhD research focused on deciphering hidden chromatin contacts from repeated genomic elements via advanced statistical profiling.
    %         \item Developed Hicberg, a novel computational tool for reconstructing missing chromatin contacts in repeated regions, extensible to pair‑ended
	% 				omics data, leading to new insights into the spatial organization of genomes.
	% 		\item  Applied Hicberg to investigate Saccharomyces cerevisiae rDNA behavior and chromatin dynamics under various stress conditions (e.g., heat,
	% 				oxidative) and kinetics.
	% 		\item  Leveraged Deep Learning to predict 2‑micron plasmid interactions with the yeast genome based on nucleosome occupancy data.
    %     \end{itemize}
    % }
	\cvitem{2016 -- 2019}{Master's Degree in Engineering (Dipl. Ing.) | \textit{Highest Honors}}{ISBS - ESIEE Paris}{}
    %     \begin{itemize}[
    %         leftmargin=1pt, 
    %         topsep=0pt,   
    %         partopsep=0pt,
    %         itemsep=-2pt,
    %         after=\vspace{-2pt},
    %         label=\textbullet 
    %         % REMOVED: align=\justified
    %     ]
    %         \justifying % <<< ADD THIS HERE
    %         \item  Led comprehensive PhD research focused on deciphering hidden chromatin contacts from repeated genomic elements via advanced statistical profiling.
    %         \item Developed Hicberg, a novel computational tool for reconstructing missing chromatin contacts in repeated regions, extensible to pair‑ended
	% 				omics data, leading to new insights into the spatial organization of genomes.
	% 		\item  Applied Hicberg to investigate Saccharomyces cerevisiae rDNA behavior and chromatin dynamics under various stress conditions (e.g., heat,
	% 				oxidative) and kinetics.
	% 		\item  Leveraged Deep Learning to predict 2‑micron plasmid interactions with the yeast genome based on nucleosome occupancy data.
    %     \end{itemize}
    % }

\end{cvtable}

\vspace*{1pt}

\cvsection{Publications}
\vspace{-5pt}
\begin{cvtable}
	\cvpubitem{Hicberg: Reconstruction of genomic signals from repeated elements - BiorXiv}{Gradit S., Ortion S., Larrous P., Koszul R., Cournac A.}
		{}{2025}
\end{cvtable}

\vspace*{1pt}

\cvsection{Selected Scientific Communication}
% \begin{cvtable}[1.85]
%     \cvcommitem{2024}{JOBIM 2024}{Toulouse, France}{Poster | Prediction of omics signals from repeated elements.}
%     \cvcommitem{2024}{JeBiF@JOBIM 2024}{Toulouse, France}{Workshop | Good practices in bioinformatics.}
%     \cvcommitem{2024}{ICTE}{St-Malo, France}{Poster | Prediction of omics signals from repeated elements.}
%     \cvcommitem{2023}{31\textsuperscript{th} Intelligent Systems For Molecular Biology}{Lyon, France}{Poster and Talk | Statistical inference of repeated sequence contacts in Hi-C maps.}
% \end{cvtable}
\vspace{-10pt}
\begin{cvtable}[1.5]
    \cvcommitem{2024}{JOBIM 2024}{Toulouse, France}{Poster | Prediction of omics signals from repeated elements.}
    \cvcommitem{2024}{JeBiF@JOBIM 2024}{Toulouse, France}{Workshop | Good practices in bioinformatics.}
    \cvcommitem{2024}{ICTE}{St-Malo, France}{Poster | Prediction of omics signals from repeated elements.}
    \cvcommitem{2023}{31\textsuperscript{th} Intelligent Systems For Molecular Biology}{Lyon, France}{Poster and Talk | Statistical inference of repeated sequence contacts in Hi-C maps.}
\end{cvtable}

\vspace*{1pt}

\cvsection{Certifications}
\vspace{-8pt}
\begin{cvtable}[1.15]
	\cvcommitem{2023}{DeepLearning.AI}{}{TensorFlow Advanced Techniques Specialization and Deep Learning Specialization}
	\cvcommitem{2023}{DeepLearning.AI - Stanford}{}{Machine Learning Specialization}
\end{cvtable}

\vspace*{1pt}

\cvsection{Honors - Awards}
\vspace{-7pt}



\begin{cvtable}
	
	\cvawarditem{2025}{1\textsuperscript{st} Prize of Read Write Grow  European SynBio Hackathon}{SynBee}{Classification of protein sequence safety through LLM and AI for Synthetic Biology designs}
	\cvawarditem{2024}{INCEPTION Progran Grant}{France}{Awarded a highly competitive grant covering the final 3 months of the PhD program demontstrating recognition of research merit andd strategic value in the field}
	\cvawarditem{2022}{1\textsuperscript{st} Prize of D4GEN Hackathon}{Genopole - Evry}{Predicting contacts between nuclear
	parasites and host genome}
\end{cvtable}

% \newpage
% \makebacksidebar
% % \newgeometry{
% % 	top=\topbottommargin,
% % 	bottom=\topbottommargin,
% % 	right=\leftrightmargin,
% % 	left=\leftrightmargin
% % }

% \cvsection{section}
% \cvsubsection{Subsection}
% \begin{cvtable}
% 	\cvitem{<dates>}{<cv-item title>}{<location>}{<optional: description>}
% \end{cvtable}

% \cvsection{cvitem}
% \cvsubsection{Multi-line with longer description}
% \begin{cvtable}
% 	\cvitem{date}{Description}{location}{Some longer and more detailed
% 		description, that takes two lines of space instead of only one.}
% 	\cvitem{date}{Description}{location}{Some longer and more detailed
% 		description, that takes two lines of space instead of only one.}
% 	\cvitem{date}{Description}{location}{Some longer and more detailed
% 		description, that takes two lines of space instead of only one.}
% \end{cvtable}

% \cvsubsection{One-line without description}
% \begin{cvtable}
% 	\cvitem{Award}{One-line description}{Sponsor}{}
% 	\cvitem{Award}{One-line description}{Sponsor}{}
% 	\cvitem{Award}{One-line description}{Sponsor}{}
% \end{cvtable}

% \cvsection{cvitemshort}
% \cvsubsection{One-line}
% \begin{cvtable}
% 	\cvitemshort{Key}{Some further description}
% 	\cvitemshort{Key}{Some further description}
% 	\cvitemshort{Key}{Some further description}
% \end{cvtable}

% \cvsubsection{Multi-line with longer description}
% \begin{cvtable}
% 	\cvitemshort{Key}{Some further description. Can fill even more than
% 		only one single line while still keeping the correct indendation level.}
% 	\cvitemshort{Key}{Some further description. Can fill even more than
% 		only one single line while still keeping the correct indendation level.}
% 	\cvitemshort{Key}{Some further description. Can fill even more than
% 		only one single line while still keeping the correct indendation level.}
% \end{cvtable}

% \cvsection{cvpubitem}
% \begin{cvtable}
% 	\cvpubitem{Publication title}{Authors}{Journal}{Year}
% 	\cvpubitem{Publication title}{Authors}{Journal}{Year}
% 	\cvpubitem{Publication title that is spanning over multiple lines and still
% 		does not look too bad}{Authors}{Journal}{Year}
% \end{cvtable}

% \cvsignature

\end{document}
